auth/facebooklogin.js

```diff
@@ -64,7 +64,8 @@ module.exports = function(app,passport) {
                     newUser.facebook.token = token; // we will save the token that facebook provides to the user
                     newUser.facebook.name  = profile.name.givenName + ' ' + profile.name.familyName; // look at the passport user profile to see how names are returned
                     newUser.facebook.email = profile.emails[0].value; // facebook can return multiple emails so we'll take the first
-
+                    var temploan ={ };
+                    newUser.loan.push(temploan);
```

Comment:
Don't push this here, we'll update all this in the controller functions. This file is just for authentication

Reply:
since loan was made array had to it i will remove it